.automation/build-srpm.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+
+# Directory, where build artifacts will be stored, should be passed as the 1st parameter
+ARTIFACTS_DIR=${1:-exported-artifacts}
+export ARTIFACTS_DIR
```

Comment:
You don't need to export the var unless you execute a shell script, where this env var is used, from this script